server/fishtest/templates/tests_view.mak

```diff
@@ -530,7 +537,28 @@
         }
         return;
       }
-      const blob = await response.blob();
+      const contentLength = parseInt(response.headers.get('Content-Length'));
+      const contentLengthFormatted = formatBytes(contentLength);
+      const reader = response.body.getReader();
+      let receivedLength = 0;
+      let chunks = [];
+
+      while (true) {
+        const { done, value } = await reader.read();
+
+        if (done) {
+          break;
+        }
+
+        chunks.push(value);
+        receivedLength += value.length;
+
+        const progressFormatted = formatBytes(receivedLength);
+        button.textContent = "Downloading... " + progressFormatted + " / " + contentLengthFormatted;
```

Comment:
The text is too wide for the button with the browser half the screen.
Dropping `"Downloading... " +` ?

Reply:
We should add a maximum width also.